charts/victoria-metrics-single/templates/server-statefulset.yaml

```diff
@@ -257,13 +257,15 @@ spec:
           configMap:
             name: {{ include "victoria-metrics.server.relabel.configname" . }}
         {{- end }}
+        {{- if or (not $app.persistentVolume.enabled) $app.persistentVolume.existingClaim }}
```

Comment:
Eventually it would be easier to understand if you define a helper variable for this on the top of the file.
- use it for this check
- use the inverted variable for the volumeClaimTemplate